db/migration/1688372371221-AddPresentationToVariables.ts

```diff
@@ -0,0 +1,91 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class AddPresentationToVariables1688372371221
+    implements MigrationInterface
+{
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(
+            `ALTER TABLE variables
+                ADD COLUMN schemaVersion INT NOT NULL DEFAULT 1,
+                ADD COLUMN processingLevel ENUM('minor', 'major'),
+                ADD COLUMN processingLog JSON,
+                ADD COLUMN titlePublic TEXT,
+                ADD COLUMN titleVariant TEXT,
+                ADD COLUMN producerShort TEXT,
+                ADD COLUMN citationInline TEXT,
+                ADD COLUMN descriptionShort TEXT,
+                ADD COLUMN descriptionFromProducer TEXT,
+                -- ADD COLUMN topicTagsLinks JSON,
+                ADD COLUMN keyInfoText JSON,
+                ADD COLUMN processingInfo TEXT,
+                ADD COLUMN licenses JSON,
+                ADD COLUMN presentationLicense JSON,
+                ADD COLUMN grapherConfigETL JSON;`
+        )
+        await queryRunner.query(
+            `ALTER TABLE datasets
+                ADD COLUMN updatePeriod INT;`
```

Comment:
An integer duration field should really have its unit as part of the name. E.g. `updatePeriodMonths`

Reply:
That makes sense, @danyx23 are you ok with this change?

Reply:
yeah good idea although what we have is `updatePeriodDays`